crates/account_manager/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+use hashsig::signature::{
+    SignatureScheme,
+    generalized_xmss::instantiations_poseidon::lifetime_2_to_the_20::winternitz::SIGWinternitzLifetime20W4,
+};
+use rand::SeedableRng;
+use rand_chacha::ChaCha20Rng;
+use sha2::{Digest, Sha256};
+use tracing::info;
+
+pub fn generate_keys(seed_phrase: &str) {
+    info!("Generating beam chain validator keys.....");
+
+    // Hash the seed phrase to get a 32-byte seed
+    let mut hasher = Sha256::new();
+    hasher.update(seed_phrase.as_bytes());
+    let seed = hasher.finalize().into();
+    info!("Seed: {:?}", seed);
```

Comment:
```suggestion
    info!("Seed: {seed:?}");
```

It is best if we inline this